libs/portals/my-pages/core/src/components/IntroWrapper/IntroWrapper.tsx

```diff
@@ -80,7 +81,9 @@ export const IntroWrapper = (props: IntroWrapperProps) => {
               {props.introComponent && (
                 <Box paddingTop={1}>{props.introComponent}</Box>
               )}
-              {props.buttonGroup}
+              <Box marginTop={4}>
+                <Inline space={2}>{props.buttonGroup}</Inline>
+              </Box>
```

Comment:
Is this responsive for mobile & tablet views?

Reply:
Kinda (at least for 2 buttons), doesn't look bad but could be better. Lets review later.